docs/introduction/apphooks.rst

```diff
@@ -21,15 +23,14 @@ This is the most basic example of an apphook for a django CMS application:
 
 .. code-block:: python
 
+    from django.utils.translation import ugettext_lazy as _
     from cms.app_base import CMSApp
     from cms.apphook_pool import apphook_pool
-    from django.utils.translation import ugettext_lazy as _
 
     class PollsApphook(CMSApp):
-        name = _("Polls Application")   # give your application a name (required)
-        urls = ["polls.urls"]           # link your app to url configuration(s)
-        app_name = "polls"
-
+        name = _("Polls Application")   # give your application a name
+        urls = ["polls.urls"]           # link it to URL configuration(s)
+        app_name = "polls"              # set the default application namespace
```

Comment:
Perhaps its a matter of preference, but I typically sort attributes alphabetically.